docs/spec/resource-scopes.md

```diff
@@ -144,4 +144,5 @@ module myMod './path/to/module.bicep' = {
 }
 ```
 
-[arm-scopes]: https://docs.microsoft.com/azure/azure-resource-manager/management/overview#understand-scope
\ No newline at end of file
+⮴📖 [resourceGroup properties] https://docs.microsoft.com/en-us/azure/templates/microsoft.resources/resourcegroups?tabs=bicep
```

Comment:
the formatting of these links are wrong. This is supposed to be referencing a link somewhere else in the file. For the link declared on L148, it is referenced on L7. If you just want to add a reference link to the end, you can do a standard markdown link like:

```
[resourceGroup properties](https://docs.microsoft.com/azure/templates/microsoft.resources/resourcegroups?tabs=bicep)
```